recipes/padel/build.sh

```diff
@@ -0,0 +1,4 @@
+mkdir -p $PREFIX/bin
+
+cp $RECIPE_DIR/padel.py $PREFIX/bin/padel
+cp -r * $PREFIX
```

Comment:
doesn't this project have a setup.py?

Reply:
You are just copying the Python file, there are a lot of ``.jar`` on this package. Shall you install all of them as well?

Reply:
Hi @marcelotrevisani, the package is written in Java. I also wrote a small python wrapper script to invoke the jar file.

Reply:
Ah ok, I see